x-pack/plugins/observability_solution/apm/server/routes/errors/get_error_groups/get_error_group_main_statistics.ts

```diff
@@ -129,16 +149,8 @@ export async function getErrorGroupMainStatistics({
             sample: {
               top_hits: {
                 size: 1,
-                _source: [
-                  TRACE_ID,
-                  ERROR_LOG_MESSAGE,
-                  ERROR_EXC_MESSAGE,
-                  ERROR_EXC_HANDLED,
-                  ERROR_EXC_TYPE,
-                  ERROR_CULPRIT,
-                  ERROR_GROUP_ID,
-                  '@timestamp',
-                ],
+                fields: [...requiredFields, ...optionalFields],
+                _source: [ERROR_LOG_MESSAGE, ERROR_EXC_MESSAGE, ERROR_EXC_HANDLED, ERROR_EXC_TYPE],
```

Comment:
why are we getting those from `_source`? It should work if we get those from `fields`

Reply:
For the same reason explained below.

Reply:
should we get `ERROR_EXCEPTION` from `_source` and everything else from `fields` based on Dario's response?

Reply:
It's basically the same thing I'm doing here, except that I'm targeting individual fields, isn't it?